### PR TITLE
Community update june2021

### PR DESCRIPTION
**Pull request by DevOpsDerek**
# About

This is a post about...

# Pre-requisites

- [*] The draft is complete and this post is ready to be reviewed.
- [*] I'm familiar with the [writing for Octopus TL;DR](https://style.octopus.com/writing-at-octopus-tldr).
- [*] The PR build passes validation, and if it doesn't, I've checked the [common validations errors](https://style.octopus.com/writing-at-octopus-tldr#common-validation-errors) and none of those apply.
- [*] The images I've included follow the [rules for working with images](https://style.octopus.com/images#rules-for-screenshots).

### CODE REVIEW

File: blog/2021-06/june-community-update/index.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+![Community Image](blogimage-community-youtube-2021.png)
+
+In this blog, I will cover the latest and greatest content from and about Octopus Deploy. We're going to publish this monthly towards the end of the month to share the latest updates with you.
+
+## Webinars
+
+In this section, I'll list out the most recent Octopus webinars.
+
+### What's new in Octopus 2020.6 and 2021.1
+
```
Comment (wordlee) [inline blog/2021-06/june-community-update/index.md:22]: Let's embed the recording here :)

File: blog/2021-06/june-community-update/index.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+- Tentacle for ARM/ARM64
+- Global Search
+- API Keys improvements
+- Project Migrations
+- New and Improved Azure App Service step
+- Github Actions for Octopus Deploy
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/NRwFdpvNYyA" frameborder="0" allowfullscreen></iframe>
+
+### Self-hosting Octopus on Linux
```
Comment (wordlee) [inline blog/2021-06/june-community-update/index.md:37]: Let's embed the second webinar here.

**Comment by wordlee** [comment]
I made some tweaks to the language to give the post that familiar voice and tone. 

I moved the update about the new webinar page into the section on webinars.

I also removed the recent blog posts - one of them was already linked to from one of the webinars, and to be honest, it feels like link to content off the blog feels like a better fit for this kind of post, but happy to discuss.

**Comment by DevOpsDerek** [comment]
I think this makes sense. The blog looks presentable, tidy and informative. I did initially think we should keep the new variable filters but that's about what's coming up than what's happened. I'm happy for this blog to be published.

File: blog/2021-07/june-community-update/index.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
+---
+title: Octopus Deploy June Community update
```
Comment (TeganAli) [inline blog/2021-07/june-community-update/index.md:2]: @DevOpsDerek are you happy for the title to be updated to "Octopus Deploy updates in June"? Not sure if "Community" has the necessary context for our readers?

File: blog/2021-07/june-community-update/index.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+- We've updated the Octopus [High Availability docs](https://octopus.com/docs/administration/high-availability) to include detailed guides for On-Premises, AWS, and Azure.
+- Our guidance for using the tools bundled with both the [AWS PowerShell](https://octopus.com/docs/deployments/custom-scripts/aws-cli-scripts) and [Azure PowerShell](https://octopus.com/docs/deployments/azure/running-azure-powershell) built-in Octopus steps has now changed. We now recommend you configure Octopus Deploy to use your versions of the required tools.
+- We've added a section to the Linux Tentacle page about [creating a Tentacle instance without requiring root or sudo privileges](https://octopus.com/docs/infrastructure/deployment-targets/linux/tentacle#rootless-instance-creation).
+
+
+## Conclusion
+
+I hope these resources are helpful. If there's any topics you’d like us to cover in our blog, or any great third-party content about Octopus Deploy you think we should include, please reach out to me at <derek.campbell@octopus.com>.
```
Comment (TeganAli) [inline blog/2021-07/june-community-update/index.md:74]: @DevOpsDerek can you please check my edits to the conclusion haven't changed the intended meaning? Thanks :)

Comment (TeganAli) [inline blog/2021-07/june-community-update/index.md:74]:
```suggestion
I hope these resources are helpful. If there are any topics you’d like us to cover in our blog, or any great third-party content about Octopus Deploy you think we should include, please reach out to me at <derek.campbell@octopus.com>.
```

File: blog/2021-07/june-community-update/index.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+- We've updated the Octopus [High Availability docs](https://octopus.com/docs/administration/high-availability) to include detailed guides for On-Premises, AWS, and Azure.
+- Our guidance for using the tools bundled with both the [AWS PowerShell](https://octopus.com/docs/deployments/custom-scripts/aws-cli-scripts) and [Azure PowerShell](https://octopus.com/docs/deployments/azure/running-azure-powershell) built-in Octopus steps has now changed. We now recommend you configure Octopus Deploy to use your versions of the required tools.
+- We've added a section to the Linux Tentacle page about [creating a Tentacle instance without requiring root or sudo privileges](https://octopus.com/docs/infrastructure/deployment-targets/linux/tentacle#rootless-instance-creation).
+
+
+## Conclusion
+
+I hope these resources are helpful. If there are any topics you’d like us to cover in our blog, or any great third-party content about Octopus Deploy you think we should include, please reach out to me at <derek.campbell@octopus.com>.
```
Comment (TeganAli) [inline blog/2021-07/june-community-update/index.md:74]: @DevOpsDerek can you please check my edits to the conclusion haven't changed the intended meaning? Thanks :)

File: blog/2021-07/june-community-update/index.md (hunk 3 above)
Comment (DevOpsDerek) [reply blog/2021-07/june-community-update/index.md:2]: This looks good to me. Thanks so much.

File: blog/2021-07/june-community-update/index.md (hunk 5 above)
Comment (DevOpsDerek) [reply blog/2021-07/june-community-update/index.md:74]: These all look great. Thanks Tegan

Comment (DevOpsDerek) [reply blog/2021-07/june-community-update/index.md:74]: This looks great

**Pull request merged**